api/v1/config.js

```diff
@@ -192,6 +192,13 @@ module.exports = {
       description:
         'Eden has bugs too. :( See what issues we have and chime in if you found a bug!',
     },
+    {
+      url: 'https://github.com/EdenServer/eden-web/issues',
+      image: 'https://static.ffxiah.com/images/icon/17001.png',
+      header: 'Website Bugs',
+      description:
+        'Eden refines rules, information, an dconnection details occassionally. Chime in if you found an error!',
```

Comment:
Maybe prefix this by saying something about "in addition to technical bugs". Because I'm sure we have them too lol

Reply:
Updated.